flink-connector-sqlserver-cdc/src/main/java/com/ververica/cdc/connectors/sqlserver/table/SqlServerTableFactory.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.sqlserver.table;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+import org.apache.flink.configuration.ReadableConfig;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.table.connector.source.DynamicTableSource;
+import org.apache.flink.table.factories.DynamicTableFactory;
+import org.apache.flink.table.factories.DynamicTableSourceFactory;
+import org.apache.flink.table.factories.FactoryUtil;
+import org.apache.flink.table.utils.TableSchemaUtils;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.ververica.cdc.debezium.table.DebeziumOptions.DEBEZIUM_OPTIONS_PREFIX;
+import static com.ververica.cdc.debezium.table.DebeziumOptions.getDebeziumProperties;
+
+/** Factory for creating configured instance of {@link SqlServerTableSource}. */
+public class SqlServerTableFactory implements DynamicTableSourceFactory {
```

Comment:
Can we have an option for configuring server time zone? I think it's necessary for configuring `RowDataDebeziumDeserializeSchema`

Reply:
> Can we have an option for configuring server time zone? I think it's necessary for configuring `RowDataDebeziumDeserializeSchema`
Yes,I will add server time zone option and configure `RowDataDebeziumDeserializeSchema`.

Reply:
Hi,@PatrickRen.Currently,Debezium does not support Spatial column types,according to class `io.debezium.connector.sqlserver.SqlServerValueConverters`.Is there need to create class `SqlServerDeserializationConverterFactory` and configure `RowDataDebeziumDeserializeSchema`'s `userDefinedConverterFactory`?